setup.py

```diff
@@ -74,7 +74,7 @@
     f"jax=={jax_version}",
     f"jaxlib=={jax_version}",
     "tomlkit;python_version<'3.11'",
-    "scipy",
+    "scipy==1.12.0",
```

Comment:
This should ideally be a `<=` since we always want to be as least restrictive as possible :)

Reply:
Yep, agree. @erick-xanadu @rmoyard can we please adjust this to be <= if we haven't already